modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/memodelrelease/v0.1.0.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/",
+      "this": "{{base}}/schemas/hbphackaton/simulation/memodelrelease/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/hbphackaton/simulation/morphology/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:MEModelReleaseShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:MEModelRelease",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:emodelRelease",
+              "name": "EModel release",
+              "description": "EModel release shape.",
+              "editorialNote": "A node constraint should be added when it is supported: {{base}}/schemas/hbphackaton/simulation/emodelrelease/v0.1.0/EModelReleaseShape",
+              "seeAlso": "{{base}}/schemas/hbphackaton/simulation/emodelrelease/v0.1.0/EModelReleaseShape",
+              "class": "nsg:EModelRelease"
+            },
+            {
+              "path": "nsg:morphologyRelease",
+              "name": "Morphology release",
+              "description": "Morphology release shape.",
+              "editorialNote": "A node constraint should be added when it is supported: {{base}}/schemas/hbphackaton/simulation/morphologyrelease/v0.1.0/shapes/MorphologyReleaseShape",
+              "seeAlso": "{{base}}/schemas/hbphackaton/simulation/morphologyrelease/v0.1.0/shapes/MorphologyReleaseShape",
+              "class": "nsg:MorphologyRelease"
+            },
+            {
+              "path": "nsg:memodelIndex",
+              "name": "Memodel index",
```

Comment:
We should agree on a coherent spelling of memodel (i.e. which letters to put capital)